app/views/providers/application_merits_task/opponent_organisations/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with(
+      model: @form,
+      url: form_path,
+      method: :patch,
+      local: true,
+    ) do |form| %>
+  <%= form.govuk_text_field(
+        :name,
+        label: { text: t(".name"), size: "m" },
+        class: "govuk-input govuk-input--width-20",
```

Comment:
Does the `govuk_text_field` helper apply the class `govuk-input` by default?
If yes, does this output the classname twice?